examples/dwarfdump.rs

```diff
@@ -134,24 +134,26 @@ fn dump_info<Endian>(file: &object::File,
     where Endian: gimli::Endianity
 {
     if let Some(debug_info) = file.get_section(".debug_info") {
-        println!(".debug_info");
-        println!("");
+        println!("\n.debug_info");
 
         let debug_info = gimli::DebugInfo::<Endian>::new(&debug_info);
 
+        let mut offset = 0;
         let mut iter = debug_info.units();
         while let Some(unit) = iter.next().expect("Should parse compilation unit") {
             let abbrevs = unit.abbreviations(debug_abbrev)
                 .expect("Error parsing abbreviations");
 
-            dump_entries(unit.entries(&abbrevs),
+            dump_entries(offset,
+                         unit.entries(&abbrevs),
                          unit.address_size(),
                          unit.format(),
                          debug_line,
                          debug_loc,
                          debug_ranges,
                          debug_str,
                          flags);
+            offset += unit.length_including_self();
```

Comment:
I think the offset stuff should be moved within the unit eventually, but doesn't need to be now.